src/main/webapp/app/exercises/quiz/manage/short-answer-question/short-answer-question-edit.component.ts

```diff
@@ -140,6 +120,37 @@ export class ShortAnswerQuestionEditComponent implements OnInit, OnChanges, Afte
         }
     }
 
+    /**
+     * Parses the text taken as parameter into text blocks
+     * @param text the text which should be parsed
+     */
+    private parseQuestionTextIntoTextBlocks(text: string): string[][] {
+        const returnValue: string[][] = [];
+        const lineText = text.split(/\n+/g);
+        lineText.forEach((line) => {
+            const textParts = this.shortAnswerQuestionUtil.divideQuestionTextIntoTextParts(line)[0];
```

Comment:
I guess we have to split the text into lines ourselves even though `divideQuestionTextIntoTextParts` already handles lines well because we have to know and add the indentation? Not as nice as I thought this solution would be, but it works correctly.

Reply:
Exactly, `divideQuestionTextIntoTextParts ` would actually do this job but as we need to know the original line for the `getIndentation ` call later on, we need to parse line break manually here.